apps/a11y-tests/src/tests/__snapshots__/Button.test.ts.snap

```diff
@@ -4,33 +4,11 @@ exports[`runs Axe on Button 1`] = `
 Array [
   Object {
     "level": "error",
-<<<<<<< HEAD
-    "locations": Array [
-      Object {
-        "annotations": Array [
-          Object {
-            "snippet": Object {
-              "text": "<div data-automation-id=\\"test\\" aria-roledescription=\\"split button\\" aria-disabled=\\"false\\" data-is-focusable=\\"true\\" role=\\"button\\" aria-haspopup=\\"true\\" aria-expanded=\\"false\\" class=\\"css-260\\" tabindex=\\"0\\">",
-            },
-          },
-        ],
-        "fullyQualifiedLogicalName": "div:nth-child(1) > .css-260[aria-roledescription=\\"split\\\\ button\\"][aria-haspopup=\\"true\\"]",
-      },
-    ],
-=======
->>>>>>> upstream/master
```

Comment:
What is up with this file coming from the `a11y-tests` app?

Reply:
Good question!